kvrocks.conf

```diff
@@ -501,7 +501,17 @@ compaction-checker-range 0-7
 #
 # rename-command KEYS ""
 
-################################ MIGRATE #####################################
+################################ MIGRATION #####################################
+# Slot migration supports two ways:
+# - redis_command: Migrate data by redis serialization protocol(RESP).
+# - raw_key_value: Migrate the raw key value data of the storage engine directly.
```

Comment:
`redis-command` and `raw-key-value`?

Reply:
I want to distinguish the item value from the item name

Reply:
I mean dash instead of underlines